extensions/npc/standalone.mjs

```diff
@@ -25,9 +35,15 @@ class Standalone {
 
     // create individual NPC
     fs.readdirSync('../run/npc').forEach((file) => {
+      if (file[0] == ".") return;
       const npcName = file.slice(0, -('.json'.length));
-      const npc = new SingleNPC(helper, npcName, `../run/npc/${file}`);
-      this.NPCs.set(npcName, npc);
+      if (npcName === "BulletinNPC") {
```

Comment:
We want to prevent hard coding `npcName` as much as possible, this way we'll have flexibility in handling multiple different bulletin NPCs.